talismanrc/.talismanrc

```diff
@@ -0,0 +1,5 @@
+fileignoreconfig:
+- filename: Foo
+  checksum: SomeCheckSum
+  ignore_detectors: []
+scopeconfig: []
```

Comment:
Please remove this file.
.talismanrc exists in .gitignore at repo root, I'm surprised how this got checked in.

Reply:
Actually, I'll let this change request go through.